frontends/web/src/routes/account/account.tsx

```diff
@@ -138,13 +139,8 @@ class Account extends Component<Props, State> {
     Promise.all(this.deviceIDs(this.props.devices).map(deviceID => {
       switch (this.props.devices[deviceID]) {
       case 'bitbox':
-        return apiGet(`devices/${deviceID}/info`)
-          .then(info => {
-            if (!info) {
-              return false;
-            }
```

Comment:
I am not 100% sure if the returned object can be undefined, seems it was added in https://github.com/digitalbitbox/bitbox-wallet-app/commit/06739c2fffdb39333fe1dc49a107f2c455b874d3

Reply:
I don't have a bb01 to make a real test on the device, but I made some checks and tested a bit using similar calls on bb02. The endpoint should always return a valid object or an error (which would be handled by `apiGet` error handler). If for any reason the call requests an invalid ID it gets a 404 error and the `then` code is not executed. I think this is safe enough.